server/controllers/user.js

```diff
@@ -46,7 +45,31 @@ class User {
       response,
       201,
       `Hello ${newUser.username}, Welcome to Author's Haven.`,
-      token
+      { token }
+    );
+  }
+
+  /**
+   * Get all non-admin users.
+   * @param {object} request
+   * @param {object} response
+   * @returns {object} The JSON response to the user.
+   */
+  static async listAll(request, response) {
+    const allUsers = await userRepo.getAllUsers();
+    if (allUsers === null) {
+      return goodHttpResponse(
+        response,
+        404,
```

Comment:
is 404 the correct status code to use here?

Reply:
`goodHttpResponse ` takes a response obj, a status code , a message and an object as the last parameter. Lets stick to that please

Reply:
> is 404 the correct status code to use here?

Yes, it is.

Reply:
@TheoOkafor 404 is not the correct status code to use in this situation. This is a valid request

Reply:
@PriscillaSam gave you the same feedback 🤦‍♂️